examples/asr/conf/vad/frame_vad_infer_postprocess.yaml

```diff
@@ -0,0 +1,38 @@
+name: &name "vad_inference_postprocessing"
+
+dataset: null # Path of json file of evaluation data. Audio files should have unique names
+num_workers: 12
+sample_rate: 16000
+evaluate: False  # whether to get AUROC and DERs, the manifest must contains groundtruth if enabled
+
+prepare_manifest:
+  auto_split: True # whether to automatically split manifest entry by split_duration to avoid potential CUDA out of memory issue.
+  split_duration: 400 # try smaller number if you still have CUDA memory issue
+
+vad:
+  model_path: "vad_multilingual_frame_marblenet" #.nemo local model path or pretrained model name or none
+  use_rttm: True # set True to output as RTTM format
+  parameters: # Parameters were tuned on 0~20db SNR noisy and clean multilingual ASR data.
+    normalize_audio_db: null  # set to non null value to normalize RMS DB of audio before preprocessing
+    window_length_in_sec: 0.0  # window length in sec for VAD context input, must be 0 for frame-VAD
+    shift_length_in_sec: 0.02 # frame-length in seconds for frame-VAD 
+    smoothing: False # Deprecated for Frame-VAD. false or type of smoothing method (eg: median, mean)
+    overlap: 0.875 # Deprecated for Frame-VAD. overlap ratio for overlapped mean/median smoothing filter. If smoothing=False, ignore this value.
+    postprocessing:
+      onset: 0.3 # onset threshold for detecting the beginning and end of a speech
```

Comment:
These are not tuned parameters, right? If we don't have a plan to tune them, how about leaving a note here?

Reply:
done